runtime/mangata-rococo/src/lib.rs

```diff
@@ -1559,6 +1559,36 @@ impl pallet_proxy::Config for Runtime {
 	type AnnouncementDepositFactor = AnnouncementDepositFactor;
 }
 
+parameter_types! {
+	// Add one item in storage and take 258 bytes
+	pub const BasicDeposit: Balance = deposit(1, 256);
+	// No item in storage but takes 66 bytes
+	pub const FieldDeposit: Balance = deposit(0, 66);
+	// No item in storage but takes 53 bytes
+	pub const SubAccountDeposit: Balance = deposit(0, 53);
+	pub const MaxSubAccounts: u32 = 100;
+	pub const MaxAdditionalFields: u32 = 100;
+	pub const MaxRegistrars: u32 = 20;
```

Comment:
Is it maximum numbers of collators identities that can be set, or i got it wrong ?

Reply:
a registrar is an entity which can pass a judgement on identity
an identity verifier of sorts
this number defines max amount of such registrars

user sets an identity -> a registrar can verify the fields to set some confidence level on the submitted fields